ReactSkia/components/RSkComponentTextInput.cpp

```diff
@@ -120,6 +120,10 @@ void RSkComponentTextInput::OnPaint(SkCanvas *canvas) {
                           std::make_shared<skia::textlayout::ParagraphBuilderImpl>(
                           paraStyle, data.layoutManager->collection_));
 
+  if (secureTextEntry_) {
+    displayString_.replace( displayString_.begin(), displayString_.end(), displayString_.size(), '*');
+  }
```

Comment:
this will replace the displaystring with the char '*' right ? 
this will effect the events emitted after this call . 
Should we take copy of the displaystring_ and then apply replace on that ? 

Also do we still need to provide size() ie., third argument for the replace call .